src/script/media/MediaDevicesHandler.ts

```diff
@@ -343,6 +359,34 @@ export class MediaDevicesHandler {
     return screenSources;
   }
 
+  /**
+   * Enumerates all known Media Devices Infos of a System.
+   * @param forceCamera If `forceCamera=true`, then an audio and video track is created to enforce the device access
+   * permissionm
+   * @returns All enumerated MediaDeviceInfos
+   */
+  private async enumerateMediaDevices(forceCamera = false): Promise<MediaDeviceInfo[]> {
```

Comment:
Actually the whole method is unnecessary. I can remove them if you want.

I would have liked to implement it like this.. I cannot use this method because otherwise we would create additional popups to ask user for permissions. So currently I check the permissions:

1. if a call is active I use the running stream or
2. I use the method .. to create a stream in the device selection, 

```typescript
streamHandler.requestMediaSreamAccess(true)
```

Reply:
I'll remove it